### PR TITLE
Show notification to login if a request fails due to failed authorization

### PR DESCRIPTION
**Pull request by lmoe**

### CODE REVIEW

File: client/client.go [hunk 1]
```diff
@@ -45,6 +48,10 @@ func processResponse(res *http.Response, decodeTo interface{}) error {
 	}
 	defer res.Body.Close()
 
+	if res.StatusCode == http.StatusUnauthorized {
```
Comment (muXxer) [inline client/client.go:50]: Why not move the `defer res.Body.Close()` to the top of the function, and then check the statuscode, and then do the `ReadAll`? Isn't that possible? Can and should we do the `defer res.Body.Close()` even if the `ReadAll` fails?

Comment (lmoe) [reply client/client.go:50]: The client will be removed soon, I've just modified it quickly to report a checkable error to the cli. Don't think we should invest more time than needed on it.

Comment (muXxer) [reply client/client.go:50]: Instead of writing that comment, you could have fixed it :trollface:

Comment (muXxer) [reply client/client.go:50]: Fixed it for you :)

**Review by muXxer — APPROVED** [review]

**Review by jorgemmsilva — APPROVED** [review]

**Pull request merged**